PhysicsTools/PatAlgos/python/slimming/slimming_cff.py

```diff
@@ -18,3 +18,25 @@
 from PhysicsTools.PatAlgos.slimming.MicroEventContent_cff import *
 from RecoEgamma.EgammaPhotonProducers.reducedEgamma_cfi  import *
 from RecoLuminosity.LumiProducer.bunchSpacingProducer_cfi import bunchSpacingProducer
+
+slimmingTask = cms.Task(
+    packedPFCandidatesTask,
+    lostTracks,
+    offlineSlimmedPrimaryVertices,
```

Comment:
this is not easily maintainable.
individual module additions better be replaced by xyzTask following the pattern of loaded files.

Maybe
```
slimmingTask = cms.Task()

 from PhysicsTools.PatAlgos.slimming.packedPFCandidates_cff import *
slimmingTask += packedPFCandidatesTask
```
and so on